### DISCUSSION TITLE
Simplify UI

### DISCUSSION THREAD

**Discussion by juancarlospaco** [UPBGE/upbge#1381] [Ideas/Features for UPBGE 0.3+]
Hello, would be nice if upbge can collapse or hide things are not directly related to BGE with an option somewhere, something like:

:heavy_check_mark: Simplify UI for Game Development

Should be some kind of Python-made boolean checkbox somewhere.

Blender keeps adding more and more stuff, and not all of them may be directly wanted for Game development.

Not remove, not delete, just visually collapsed by 1 opt-in checkbox.

**Comment by juancarlospaco** [comment]
Mmm...  no one reads this?. :thinking:

**Reply by lordloki** [reply]
Yes, we read it. The issue here is that to collapse or hide unused panels we need to do a lot modifications on blender UI files. Usually we do the UPBGE UI modifications in our own UI file. If we start to modify all the Blender UI files to collapse/hide then surely we will have more problems during the dayly merges.

We need to investigate how can do that with minimum Blender UI code modification

**Reply by juancarlospaco** [reply]
The state is serialized into a config file somewhere, so maybe change that config file instead of patching the Python code directly ?.
